interface/reports/daily_summary_report.php

```diff
@@ -0,0 +1,409 @@
+<?php
+/*
+ *  Daily Summary Report. (/interface/reports/daily_summary_report.php
+ *  
+ *
+ *  This report shows date wise numbers of the Appointments Scheduled,
+ *  New Patients, Visited patients, Total Charges, Total Co-pay and Balance amount for the selected facility.
+ * 
+ * Copyright (C) ....
+ * 
+ * LICENSE: This program is free software; you can redistribute it and/or 
+ * modify it under the terms of the GNU General Public License 
+ * as published by the Free Software Foundation; either version 3 
+ * of the License, or (at your option) any later version. 
+ * This program is distributed in the hope that it will be useful, 
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of 
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the 
+ * GNU General Public License for more details. 
+ * You should have received a copy of the GNU General Public License 
+ * along with this program. If not, see <http://opensource.org/licenses/gpl-license.php>;. 
+ * 
+ * @package OpenEMR 
+ * @author Rishabh Software
+ * @link http://www.open-emr.org 
+ *
+ */
+
+$fake_register_globals = false;
+$sanitize_all_escapes = true;
+
+require_once("../globals.php");
+require_once "$srcdir/options.inc.php";
+require_once "$srcdir/appointments.inc.php";
```

Comment:
Do note I was unable to bring in your changes to appointments.inc.php since you are working on an out of date script. Recommend working instead inside a openemr repository. See here on how to set up an OpenEMR repository:
(wiki is now down; I will post the link later TODO )